docs/modules/user-guide/pages/adding-a-kubernetes-or-openshift-component-to-a-devfile.adoc

```diff
@@ -0,0 +1,6 @@
+:description: Adding a kubernetes or openshift component to a devfile
+:navtitle: Adding a Kubernetes or Openshift component
```

Comment:
OpenShift

Are the `navtitle` and `description` attributes supposed to contain the same information? If so, we might want to change the capitalization there as well. :)